pytest/lib/cluster.py

```diff
@@ -408,6 +409,7 @@ def init_cluster(num_nodes, num_observers, num_shards, config, genesis_config_ch
                  for line in err.decode('utf8').split('\n') if '/test' in line]
     assert len(node_dirs) == num_nodes + num_observers, "node dirs: %s num_nodes: %s num_observers: %s" % (len(node_dirs), num_nodes, num_observers)
 
+    print("Search for stdout and stderr in %s" % node_dirs)
```

Comment:
remove?

Reply:
why remove?

Reply:
Oh I thought it is some debug output. Nvm